### PR TITLE
Fix #307 Load Akhand Paath View correctly

### LINKED ISSUE #307: Akhand Paatth View infinite loop
When Akhand Paatth View is on, loading a Shabad keeps loading more and more Angs without waiting for scroll

### PR DESCRIPTION
**Pull request by tsingh777**

### CODE REVIEW

File: www/js/viewer.js [hunk 1]
```diff
@@ -293,7 +297,6 @@ const showLine = (ShabadID, LineID) => {
     } else {
       smoothScroll(`#apv #slide${LineID}`);
     }
-    castShabadLine(LineID);
```
[the remark this answers is not in the document]
Comment (tsingh777) [reply www/js/viewer.js]: Right now APV doesn't work with casting (https://github.com/KhalisFoundation/sttm-desktop/issues/282). 

We need to change up the chrome app a bit to get APV to work. Don't think its coded to handle APV at the moment (https://github.com/KhalisFoundation/sttm-chromecast/blob/master/src/js/index.js). 

Should we disable APV when casting begins?

[the remark this answers is not in the document]
Comment (ManjotS) [reply www/js/viewer.js]: If its   broken we should disable  the ability and in the future make  this  work.

[the remark this answers is not in the document]
Comment (navdeepsinghkhalsa) [reply www/js/viewer.js]: @ManjotS Which ability should be disabled, Singh Jee? APV or casting?

**Comment by ManjotS** [comment]
I  think have a pop  up  and  say “Chromecast is not available with  Akhand
Patt  View in this  version.  (Coming  soon)”  and disable  chromecast for
apv

**Comment by navdeepsinghkhalsa** [comment]
@saintsoldierx @maneetpaul Singh Jees, could you please weigh in?

File: www/js/viewer.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (tsingh777) [reply www/js/viewer.js]: Disabling APV.

**Comment by saintsoldierx** [comment]
Not ideal but put a message like mentioned somewhere on the presenter window - dont show the message on output.

**Comment by tsingh777** [comment]
I was thinking of adding to the label in next to the toggle. 
image
Maybe put something like "Akhand Paatth View (Beta - No Casting)" <- could use some help with words. needs to be short so it fits properly.

**Comment by saintsoldierx** [comment]
How about "Akhand Paatth View (Beta) (ChromeCast Unavailable)"

**Comment by tsingh777** [comment]
@saintsoldierx this ok?
image

**Pull request merged**